ansible/configs/ocp-workshop/env_vars.yml

```diff
@@ -147,12 +147,19 @@ common_packages:
   - at
   - sysstat
   - strace
+  - net-tools
+  - iptables-services
```

Comment:
just wondering: could there be a conflict between firewalld and iptables-services ?